rpms/CentOS/6/i386/Dockerfile

```diff
@@ -7,10 +7,37 @@ RUN yum -y install util-linux-ng centos-release-scl \
     gcc-multilib make git openssh-clients rpm-build \
     sudo gnupg automake autoconf libtool \
     policycoreutils-python yum-utils epel-release \
-    redhat-rpm-config rpm-devel
+    redhat-rpm-config rpm-devel autopoint gettext \
+    zlib zlib-devel nspr nspr-devel \
+    nss nss-devel kenel-headers magic magic-devel \
+    db4 db4-devel zlib zlib-devel rpm-build bison \
```

Comment:
Zlib and zlib-devel are twice, rpm-build too.

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/378/commits/90204700decee13e474b57e790795ab160dc1a52